npm-shrinkwrap.json

```diff
@@ -14,13 +14,44 @@
         "@jop-software/astro-cookieconsent": "^3.0.1",
         "astro": "^5.13.9",
         "astro-broken-links-checker": "imazen/astro-broken-link-checker",
+        "astro-mermaid": "^1.1.0",
+        "mermaid": "^11.12.1",
         "rehype-external-links": "^3.0.0",
         "rehype-figure-title": "^1.0.0",
+        "rehype-mermaid": "^3.0.0",
         "sharp": "~0.34.4",
         "starlight-links-validator": "^0.18.0",
         "typescript": "^5"
       }
     },
+    "node_modules/@antfu/install-pkg": {
```

Comment:
This concerns me, as it's ripe for abuse by _any other_ package even if mermaid is using it wisely.  I do question the why a diagramming tool needs this in the first place.